mappings/net/minecraft/util/WorldSavePath.mapping

```diff
@@ -1,6 +1,30 @@
 CLASS net/minecraft/class_5218 net/minecraft/util/WorldSavePath
+	COMMENT A collection of constants that contain the path of key files and folders in a save directory.
+	FIELD field_24181 Lnet/minecraft/class_5218;
+		COMMENT Stores the statistics of players that have played on the world.
+		COMMENT {@since 1.7.2}
```

Comment:
No need for this, we didn't exist back then, and everything in the javadoc is specific to the current Minecraft version.